config/database.yml

```diff
@@ -24,6 +24,7 @@ default: &default
 development:
   <<: *default
   database: askdarcel_development
+  url: <%= ENV['DATABASE_URL'] %>
```

Comment:
Yes, it just makes it explicit. It also makes it so that the development environment parallels both the test and production environments.